xwiki-platform-core/xwiki-platform-flamingo/xwiki-platform-flamingo-themes/xwiki-platform-flamingo-theme-ui/src/main/resources/FlamingoThemesCode/Translations.xml

```diff
@@ -107,7 +107,7 @@ platform.flamingo.themes.category.panels=Panels
 platform.flamingo.themes.category.breadcrumb=Breadcrumb
 platform.flamingo.themes.category.advanced=Advanced
 platform.flamingo.themes.defaultvalue=Default
-platform.flamingo.themes.lessCode.description=In this field, you can directly write LESS code that will be added into the skin.
+platform.flamingo.themes.lessCode.description=In this field, you can directly write new LESS code for the skin.
```

Comment:
It was better before. Please revert this change.

Reply:
Okay